src/Robot.Drivers/Switches/SwitchSensorEventArgs.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Robot.Drivers.Switches
+{
+    public class SwitchSensorEventArgs
+    {
+        /// <summary>
+        /// Switch Unique Identifier.
+        /// </summary>
+        public int SwitchId { get; set; }
+
+        /// <summary>
+        /// Switch position
+        /// </summary>        
+        public bool On { get; set; }
```

Comment:
Is there any reason why the SwitchMessage has an On property and the ISwitchSensor one named IsOn?

Reply:
Semantically is no difference :)
I was just thinking that EventArg is more related to the state transfer and is like a flag 'ON',  SwitchSensor on the other hand has a real exposed property => IsOn.

But you are right, there is no real reason and is confusing. I am going to make it **IsOn** to keep consistent.